Dockerfile

```diff
@@ -2,9 +2,12 @@ FROM deriv/dzil
 ARG HTTP_PROXY
 
 WORKDIR /app
-ONBUILD COPY . /app/
+# Conditional copy - we want whichever files exist, and we'd typically expect to see at least one
+ONBUILD COPY aptfil[e] cpanfil[e] dist.in[i] /app/
```

Comment:
why we also have `/app/` here at this step? Especially that this is what we moved to its own step below.

Reply:
because it's the working directory, and we don't need two copies of the same files in different places